lib/core/src/lib/notifications/components/notification-history.component.scss

```diff
@@ -69,3 +69,8 @@
         padding: 0;
     }
 }
+
+.mat-list-base .mat-list-item.adf-notification-history-menu-no-message .mat-line,
+.mat-list-base .mat-list-item .adf-notification-history-menu-message.mat-line {
```

Comment:
```suggestion
.mat-list-base .mat-list-item .adf-notification-history-menu-no-message .mat-line,
.mat-list-base .mat-list-item .adf-notification-history-menu-message .mat-line {
```

Reply:
These is how it is supposed to be, this is the selector of where I need to make the change

Reply:
@nikita-web-ua please have a look at my comments here in Jatin's PR where you commented space for selectors too. I explained there why sometimes we need space and sometimes no for selectors :) 
https://github.com/Alfresco/alfresco-content-app/pull/3018#discussion_r1118517484